### DISCUSSION TITLE
Why can't update an Enum field with generated updateResolver?

### DISCUSSION THREAD

**Discussion by ChemaCLi** [MichalLytek/typegraphql-prisma#155] [Q&A]
Hi there.

I'm trying this generator for the first time and I noticed that the generated Input for an enum field for an update operation is defined as `NullableEnumRoleFieldUpdateOperationsInput`.

My question is WHY is this defined so? I mean, It doesn't make me any sense 🤔  I would like to know the reason or if this is an error. Thanks!

**Comment by MichalLytek** [comment]
It's defined like this because that's how Prisma defines that in DMMF.

Please elaborate on a higher level about your issue. Is it related to #32?
